docs/docs/legal/privacy-policy.mdx

```diff
@@ -6,25 +6,26 @@ import PiidTable from './_piid-table.mdx';
 
 ## Introduction
 
-This privacy policy applies to CAOS Ltd., the websites it operates (including caos.ch, zitadel.ch, zitadel.cloud and zitadel.com) and the services and products it provides (including ZITADEL). This privacy policy describes how we process personal data for the provision of this websites and our products.
+This privacy policy applies to CAOS Ltd., the websites it operates (including zitadel.ch, zitadel.cloud and zitadel.com) and the services and products it provides (including ZITADEL). This privacy policy describes how we process personal data for the provision of this websites and our products.
 
 If any inconsistencies arise between this Privacy Policy and the otherwise applicable contractual terms, framework agreement, or general terms of service, the provisions of this Privacy Policy shall prevail. This privacy policy covers both existing personal data and personal data collected from you in the future.
 
-The responsible party for the data processing described in this privacy policy is:
+The responsible party  for the data processing described in this privacy policy and contact for questions and issues regarding data protection is
 
-CAOS AG  
-Teufener Strasse 19  
-9000 St. Gallen  
-SWITZERLAND
-Contact for questions and issues regarding data protection is:
-
-CAOS AG  
+**CAOS AG**  
 Data Protection Officer  
 Teufener Strasse 19  
 9000 St. Gallen  
```

Comment:
Does this not need a name of a natural person?

Reply:
For the contact part, yes. Processor is the company. I had it split before, is it clearer then?
We could remove the contact. Personally I like the fact that we state a contact person/address.

Reply:
Ah, I see what you did there.

Let's keep it in this case